src/relay/op/tensor/transform.cc

```diff
@@ -891,6 +891,120 @@ RELAY_REGISTER_OP("broadcast_to_like")
 .add_argument("broadcast_type", "Tensor", "Provide the type to broadcast to.")
 .set_support_level(10)
 .add_type_rel("BroadCastToLike", BroadCastToLikeRel);
+// strided_slice
+TVM_REGISTER_NODE_TYPE(StridedSliceAttrs);
+bool StridedSliceRel(const Array<Type>& types,
+                 int num_inputs,
+                 const Attrs& attrs,
+                 const TypeReporter& reporter) {
+  CHECK_EQ(types.size(), 2);
+  const auto* data = types[0].as<TensorTypeNode>();
+  CHECK(data != nullptr);
+  if (data->shape.size() == 0) return false;
+
+  const StridedSliceAttrs *param = attrs.as<StridedSliceAttrs>();
+  CHECK(param != nullptr);
+
+  auto dshape = data->shape;
+  auto num_axis = dshape.size();
+
+  std::vector<IndexExpr> begin_vec;
+  for (auto i : param->begin) {
+    begin_vec.push_back(i);
+  }
+  for (auto i = begin_vec.size(); i < num_axis; ++i) {
+    begin_vec.push_back(0);
+  }
+
+  std::vector<IndexExpr> end_vec;
+  for (auto i : param->end) {
+    end_vec.push_back(i);
+  }
+  for (auto i = end_vec.size(); i < num_axis; ++i) {
+    end_vec.push_back(dshape[i]);
+  }
+
+  std::vector<IndexExpr> stride_vec;
+  for (auto i : param->stride) {
+    stride_vec.push_back(i);
+  }
+  for (auto i = stride_vec.size(); i < num_axis; ++i) {
+    stride_vec.push_back(1);
+  }
+  std::vector<IndexExpr> oshape(dshape.size());
+
+  for (size_t i = 0; i < num_axis; ++i) {
+      auto begin_range = reporter->Assert(stride_vec[i] < 0) ? -1 : 0;
```

Comment:
using Assert here is not reliable, as if stride_vec is symbolic, then assert does not reflect anything.